jane/build-resolved-files-for-ci

```diff
@@ -22,6 +23,21 @@ dune_targets=$(
     echo _build/default/.ocamlcommon.objs/byte/$cmi.cmi
   done
 )
+
+# ocamlbytecomp mlis
+mlis=$(
+  { echo driver/{errors,compile,maindriver}.mli
+  } |
+    tr ' ' '\n'
+)
+echo "$mlis"
+dune_targets=$(
+  for mli in $mlis; do
+    cmi=$(basename ${mli%.mli})
+    echo _build/default/.ocamlbytecomp.objs/byte/$cmi.cmi
+  done
+)
+
```

Comment:
This currently overwrites the value of `dune_targets` which means that the earlier ones won't be built. How about appending `$dune_targets` to the front? (We should do the same with `mlis`.)